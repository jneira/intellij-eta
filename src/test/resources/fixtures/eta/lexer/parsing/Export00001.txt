ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Export00001')
WHITE_SPACE ('\n  ')
IToparen ('(')
WHITE_SPACE ('\n    ')
IToparen ('(')
ITqvarsym ('Prelude.$')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
IToparen ('(')
ITqvarsym ('Prelude.$!')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
IToparen ('(')
ITqvarsym ('Control.Category..')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
ITqvarid ('Prelude.not')
WHITE_SPACE ('\n  ')
ITcparen (')')
WHITE_SPACE ('\n')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')