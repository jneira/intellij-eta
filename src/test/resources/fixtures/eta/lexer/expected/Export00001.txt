ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Export0000')
WHITE_SPACE ('1\n  ')
IToparen ('(')
WHITE_SPACE ('\n    ')
IToparen ('(')
WHITE_SPACE ('')
ITqvarsym ('Prelude.$')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
IToparen ('(')
WHITE_SPACE ('')
ITqvarsym ('Prelude.$!')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
IToparen ('(')
WHITE_SPACE ('')
ITqvarsym ('Control.Category..')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('\n  ')
ITcomma (',')
WHITE_SPACE (' ')
ITqvarid ('Prelude.not')
WHITE_SPACE ('\n  ')
ITcparen (')')
WHITE_SPACE ('\n')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')